soh/soh/OTRGlobals.cpp

```diff
@@ -336,6 +336,7 @@ OTRGlobals::OTRGlobals() {
 
     gSaveStateMgr = std::make_shared<SaveStateMgr>();
     gRandoContext = Rando::Context::CreateInstance();
+    Rando::Context::GetInstance()->InitStaticData();
```

Comment:
```suggestion
    gRandoContext->InitStaticData();
```
To make it similar to the lines below.